From: Sebastian Ramacher <sramacher@debian.org>
Date: Wed, 19 Apr 2017 23:02:17 +0200
Subject: Link with xcb and X libraries to fix underlinking

---
 libobs/CMakeLists.txt | 13 +++++++++----
 1 file changed, 9 insertions(+), 4 deletions(-)

diff --git a/libobs/CMakeLists.txt b/libobs/CMakeLists.txt
index 6840e4f..c386102 100644
--- a/libobs/CMakeLists.txt
+++ b/libobs/CMakeLists.txt
@@ -13,8 +13,8 @@ endif()
 
 if(UNIX)
 	if (NOT APPLE)
-		find_package(X11_XCB REQUIRED)
-		find_package(XCB OPTIONAL_COMPONENTS XINPUT)
+		find_package(XCB COMPONENTS XCB REQUIRED OPTIONAL_COMPONENTS XINPUT)
+		find_package(XII REQUIRED)
 		if (XCB_XINPUT_FOUND)
 			set(USE_XINPUT "1")
 		else()
@@ -198,12 +198,17 @@ elseif(UNIX)
 	endif()
 
 	include_directories(
-		${X11_XCB_INCLUDE_DIRS})
+		${XCB_INCLUDE_DIRS}
+		${X11_XCB_INCLUDE_DIRS}
+		${X11_X11_INCLUDE_DIRS})
 	add_definitions(
+		${XCB_DEFINITIONS}
 		${X11_XCB_DEFINITIONS})
 	set(libobs_PLATFORM_DEPS
 		${libobs_PLATFORM_DEPS}
-		${X11_XCB_LIBRARIES})
+		${XCB_LIBRARIES}
+		${X11_XCB_LIBRARIES}
+		${X11_X11_LIB})
 
 	if(USE_XINPUT)
 		include_directories(
